src/components/NewComment/NewComment.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { FC } from 'react';
+import { Comment } from '../../types';
+import './NewComment.css';
+
+interface Props {
+  comment: Comment;
+}
+
+export const NewComment: FC<Props> = ({ comment }) => {
+  const { name, body, email } = comment;
+
+  return (
+    <div className="comment">
+      <div className="comment__name">{name}</div>
+      <div className="comment__body">{body}</div>
```

Comment:
```suggestion
      <p className="comment__body">{body}</p>
```
Если внутри лежит только текст, зачем `<div>`?